e2etest/declarativeScenario.go

```diff
@@ -43,6 +43,7 @@ type scenario struct {
 	operation           Operation
 	validate            Validate
 	fromTo              common.FromTo
+	credTypes           [2]common.CredentialType
```

Comment:
Could have had an option bag like thing here. I dont think there will be many changes here. But any change here, we'llhave to change a lot of files.